cb_hr_views/__manifest__.py

```diff
@@ -5,14 +5,15 @@
     "name": "Cb Hr Views",
     "summary": """
         Views for HR modules in Creu Blanca""",
-    "version": "14.0.1.0.0",
+    "version": "16.0.1.0.0",
     "license": "AGPL-3",
     "author": "CreuBlanca",
     "website": "https://github.com/tegin/cb-hr",
     "depends": [
         "base_fontawesome",
         "base_iban",
         "base_report_to_printer",
+        "medical_base",
```

Comment:
This was not needed, as we already have medical_administration_center. However, it is not a real issue